docs/UserGuide.md

```diff
@@ -524,6 +528,24 @@ Format: `matchBuyer FULL_NAME`
 
 <div style="page-break-after: always;"></div>
 
+[//]: # (@@author redcolorbicycle)
```

Comment:
Missing ending author tag

Reply:
I agree, this would cause @redcolorbicycle to be the author for all the codes below.